src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -44,6 +45,10 @@ class SessionService(
   private val policyNonAssociationWholeDay: Boolean,
 ) {
 
+  companion object {
+    val LOG: Logger = LoggerFactory.getLogger(this::class.java)
```

Comment:
Minor observation, we could rename this val to log to keep it consistent with other instances.

Reply:
Not sure as it is a constant and the convention with constants is uppercase right?

Reply:
Ok, yes as mentioned a very minor observation.